katsdpdata/utilities.py

```diff
@@ -0,0 +1,83 @@
+import logging
```

Comment:
Nice, I really like this utilities file. I stole this boto dict code from katdal at some point i think, so now there are probably 3 repos that have the same code in it. So it might be worth discussing whether we should make a standalone s3utilities module that we can all use. It could include a set of canned s3 bucket policies that are used in different libraries.

Reply:
Oh, yeah, I didn't realise that this code was used (duplicated) in so many places. I'll make a ticket to remind us to think about this. (My knee jerk reaction would be to say the bucket manager is probably the most logical place to put it, but let's think about that)